custom_components/tuya_local/devices/lsc_smart_ceiling_light.yaml

```diff
@@ -0,0 +1,228 @@
+name: CCT + RGB Ceiling Light
+products:
+  - id: bwyutsrcsnwf3p5o
+    name: LSC Smart Connect Smart Ceiling Light
+
+primary_entity:
+  entity: light
+  name: Main light
+  icon: mdi:circle
+  dps:
+    - id: 63
+      type: boolean
+      name: switch
+    - id: 22
+      name: brightness
+      type: integer
+      range:
+        min: 10
+        max: 1000
+    - id: 23
+      name: color_temp
+      type: integer
+      range:
+        min: 0
+        max: 1000
+      mapping:
+        - target_range:
+            min: 2700
+            max: 6500
+
+secondary_entities:
+  - entity: light
+    name: Halo
+    icon: "mdi:circle-outline"
+    dps:
+      - id: 54
+        type: boolean
+        name: switch
+      - id: 24
+        name: rgbhsv
+        type: hex
+        format:
+          - name: h
+            bytes: 2
+            range:
+              min: 0
+              max: 360
+          - name: s
+            bytes: 2
+            range:
+              min: 0
+              max: 1000
+          - name: v
+            bytes: 2
+            range:
+              min: 0
+              max: 1000
+      - id: 24
+        type: hex
+        name: brightness
+        range:
+          min: 0
+          max: 1000
+        mapping:
+          - mask: "00000000FFFF"
+
+  - entity: switch
```

Comment:
This might be better as a select with translation key initial_state, and mappings true: "off", false: "on" (or false: memory, depending on actual behaviour)

Reply:
Makes sense, will try this and update the pull req.

Reply:
Changed. Had to change it from a boolean to a select. "initial_state" doesn't work on a boolean.